src/test/scala/akka/wamp/ManagerSpec.scala

```diff
@@ -56,7 +58,46 @@ class ManagerSpec
     pending
   }
 
-  it should "Fail on wrong message format" in { f =>
+  it should "Receive text messages and emit Events" in { f =>
```

Comment:
Please let's move this discussion to issue #31 as I believe it's a matter of configuration